StackReviewSuggestedEditsRework.user.js

```diff
@@ -237,11 +237,11 @@
         removeElement(`#${diffChoices}`);
 
         const choices = document.querySelector(filterDiff); // ".js-diff-choices"
-        if (choices) {
-            moveToFilterLine(choices, true);
-            choices.firstElementChild.style.padding = "10px";  // the larger padding on one takes precedence
-            choices.id = diffChoices;
-        }
+        if (!choices) return;
+
+        moveToFilterLine(choices, true);
+        choices.id = diffChoices;
+        [...choices.children].forEach((button) => button.classList.remove("s-btn__xs"));
```

Comment:
I tried this with buttons and not moving the progress.  ["s-btn__xs" and padding](https://i.stack.imgur.com/Yjpyt.png) seems do-able to me. [No "s-btn__xs" even without padding](https://i.stack.imgur.com/OJZQ5.png) is a bit too cramped up.